### PR TITLE
Adding rax auth

### PR DESCRIPTION
**Pull request by rnirmal**
This will support rax auth and auth 1.1 too.

### CODE REVIEW

**Comment by ed-** [comment]
+1

**Comment by TimSimpsonR** [comment]
Supporting our own auth system... what a novel concept!
+1

File: reddwarfclient/client.py [hunk 1]
```diff
@@ -91,6 +93,32 @@ def basic_auth(self):
                 return
         raise nova_exceptions.EndpointNotFound()
 
+    def _rax_auth(self):
+        """Authenticate against the Rackspace auth service."""
+        body = {"auth": {
```
Comment (cp16net) [inline reddwarfclient/client.py]: single quotes for keys :)

Comment (imsplitbit) [inline reddwarfclient/client.py]: oh you better pray you never do that :-)

**Comment by imsplitbit** [comment]
+1 I'm good either way

File: reddwarfclient/client.py (hunk 1 above)
Comment (rnirmal) [inline reddwarfclient/client.py]: Done

**Comment by hub-cap** [comment]
plus ohne

**Pull request merged**